src/components/common/Questionnaire/ExpandedModal/index.js

```diff
@@ -40,11 +48,9 @@ const ExpandedModal = ({ handleToggleModalOpen }) => {
   }, [dispatch, setLocalStorageValue, userFeedback]);
 
   const handleNext = useCallback(() => {
-    if (isLastQuestion) {
-      handleSubmit();
-    }
-    handleNextStep();
-  }, [handleSubmit, isLastQuestion]);
+    if (isLastQuestion) handleSubmit();
+    if (isEnabled) handleNextStep();
```

Comment:
承上，如此一來就可以不用在這裡加上條件，因為只有 `isEnabled` 的時候才能執行這個
```suggestion
    handleNextStep();
```